Designs/UI/css/styling.css

```diff
@@ -0,0 +1,203 @@
+margin-top: -140px;
+
+}
+#event-heading-2{
+    text-align: center;
+    font-size: 50px;
+    font-family: 'Segoe UI', 'Open Sans', 'Helvetica Neue', 'sans-serif';
+    text-shadow: 2px 2px 2px #5a3f3f;
+}
+/* Styling the text and password boxes */
+input[type=text], input[type=password] {
+	width: 80%;
+	padding :6px 20px;
+	margin: 8px 0px;
+	display:inline-block; 
+	font-size:18px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+    font-family: 'Segoe UI', 'Open Sans', 'Helvetica Neue', 'sans-serif';
+
+}
+/* Styling the buttons(Submit and Login) */
+button {
+	padding: 6px 20px;
+	margin: 8px 0px;
+	display: inline-block;;
+	font-size: 18px; 
+	background-color: #d33b36;
+    cursor: pointer; 
+    width: 40%;
+    border-radius: 4px;
+    font-family: 'Segoe UI', 'Open Sans', 'Helvetica Neue', 'sans-serif';
+}
+button:hover {
+	opacity: 0.8;
+}
+/*Styling login button*/
+ #btn-login {
+	width: 80%;
+}
+/*Styling sign up button*/
+#btn-signup{
+	width: 80%;
+}
+/*Center content*/
+.add-event, .event-table{
+    margin: auto;
+    padding: 10px;
+    width: 80%;
+}
+
+table {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+    margin-top: 10px;
+}
+td, th {
+    border: 1px solid #dddddd;
+    text-align: left;
+    padding: 10px;
+}
+
+tr:nth-child(even) {
+    background-color: #dddddd;
+}
+/*Style add button*/
+.btn-add-event{
+    background-color: #d33b36;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+}
+/*Navigation*/
+.nav {
+	height: 50px;
+	margin-right: 10px;
+	float: right;
+    font-size: 18px;
+}
+
+.nav ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.nav ul li {
+	display: inline-block;	
+}
+
+.nav ul li a {
+    display: block;
+    padding: 10px;
+    text-align: center;
+    text-decoration: none;
+    color: #5a3f3f;
+    font-family: 'Segoe UI', 'Open Sans', 'Helvetica Neue', 'sans-serif';
+}
+
+.nav ul li a:hover {
+    color: #d33b36;
+}
+
+#nav > ul > a {
+	padding-right: 8px;
+}
+
+.clear {
+	clear: both;
+}
+#content {
+	width: 100%;
+    height: 100px;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: auto;
+	text-align: center;
+	align-content: center;
+}
+#event-heading-1 {
+	margin-top: -50px;
+	font-size: 50px;
+    font-family: 'Segoe UI', 'Open Sans', 'Helvetica Neue', 'sans-serif';
+    color: #d33b36;
+    text-shadow: 2px 2px 2px #5a3f3f;
+}
+#event-index-p{
+    font-size: 15px;
+    color: #5a3f3f;
+    font-family: 'Segoe UI', 'Open Sans', 'Helvetica Neue', 'sans-serif';
+}
+footer {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    height: 50px;
+    width: 100%;
+    overflow:hidden;
+}
+
+#footer-wrapper {
+	width: 100%;
```

Comment:
Try to align all statements to be on same level for consistency

Reply:
Thanks, will look into that Sir. Anything that I can add to the UI for improvement? Designwise...